features/claim-mgt/org.wso2.carbon.claim.mgt.server.feature/resources/conf/claim-config.xml

```diff
@@ -549,6 +549,14 @@
 				<AttributeID>preferredChannel</AttributeID>
 				<Description>Preferred Notification Channel</Description>
 			</Claim>
+			<Claim>
+				<ClaimURI>http://wso2.org/claims/identity/accountConfirmedTime</ClaimURI>
+				<DisplayName>Confirmed Time</DisplayName>
+				<!-- If user store based identity store is used, a proper mapped attribute
+				in your user store must be configured for this. -->
+				<AttributeID>confirmedTime</AttributeID>
```

Comment:
If this is not supported in the embedded ldap we need to add to add it.

Reply:
Addressed with https://github.com/wso2-extensions/identity-userstore-ldap/pull/40.